### PR TITLE
added resume button to view resume

### PR DESCRIPTION
**Pull request by TTRIFIC**

### CODE REVIEW

File: src/components/Navbar/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect } from "react";
 import { FaBars } from "react-icons/fa";
 import { IconContext } from "react-icons/lib";
 import { animateScroll as scroll } from "react-scroll";
+import resume from "../../Files/MatthewBeckermanResumeSP2022.pdf";
```
Comment (KrastanD) [inline src/components/Navbar/index.js:5]: Instead of `Files`, call the folder `assets`

Comment (TTRIFIC) [reply src/components/Navbar/index.js:5]: Should I restructure the other folders as well? Currently have separate folders for my images and videos

Comment (KrastanD) [reply src/components/Navbar/index.js:5]: Yes. One `assets` folder. And within it you can separate it out into images, videos or just top level. However you like.

Comment (TTRIFIC) [reply src/components/Navbar/index.js:5]: Gotcha, so then do I close this pull request and then create a new one after I've made the changes?

Comment (KrastanD) [reply src/components/Navbar/index.js:5]: Nope. When you push up an update it will get added to this PR. It becomes like a history for this feature.

Comment (TTRIFIC) [reply src/components/Navbar/index.js:5]: Ahhhh, that's pretty neat

**Review by KrastanD — APPROVED** [review]
Looks much better!

**Pull request merged**